src/main/resources/public/jsx/geo-dash-widget-editor.js

```diff
@@ -19,59 +19,48 @@ class BasicLayout extends React.PureComponent{
          graphReducer: "Min"
```

Comment:
onLayoutChange does not appear to be used from the state.  It is not necessary to store a function in the state